src/sections/Menu.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Manager, Target, Popper } from 'react-popper';
+import Avatar from 'material-ui/Avatar';
+import Icon from 'material-ui/Icon';
+import Button from 'material-ui/Button';
+import Grow from 'material-ui/transitions/Grow';
+import Paper from 'material-ui/Paper';
+import { MenuItem, MenuList } from 'material-ui/Menu';
+import { withStyles } from 'material-ui/styles';
+import ClickAwayListener from 'material-ui/utils/ClickAwayListener';
+import { Link } from 'react-router-dom';
+
+import './Menu.css';
+
+const styles = theme => ({
+  root: {
+    display: 'flex'
+  },
+  paper: {
+    marginRight: theme.spacing.unit * 2
+  },
+  popperClose: {
+    pointerEvents: 'none'
+  }
+});
+
+class MenuListComposition extends React.Component {
```

Comment:
I think this is more complicated than we need, and I'm a fan of just using the simpler material-ui components when they exist instead of integrating with React-popper. I personally find the [change-transition example](https://material-ui-next.com/demos/menus/#change-transition) a lot simpler to reason about and probably a lot simpler to maintain.

Reply:
I updated the menu to [change-transition example](https://material-ui-next.com/demos/menus/#change-transition).  However for some reason I could not import the 'material-ui/Fade' transition component locally but the menu still functions without it.